src/components/molecules/UserProfilePicture/UserProfilePicture.scss

```diff
@@ -1,83 +1,152 @@
 @import "scss/constants.scss";
 
 $fadedWhite: rgba(255, 255, 255, 0.8);
-$borderWidth: 5px;
 
-.profile-picture-container {
-  position: relative;
+// TODO: this was copied from Audience.scss (do we want to move the variable to constants?)
+$avatarSize: max(4vh, 3vw);
+
+// TODO: should we move these to using CSS --variables rather than these SCSS vars
+$reactionContainerOffset: calc(-1 * #{$avatarSize});
+$reactionSize: calc(#{$avatarSize / 1.5});
+$reactionOffset: calc(-1 * #{$reactionSize} / 2);
+$shoutBounceTranslateXOffset: calc(#{$avatarSize} / 2);
```

Comment:
[TODO] should we move these to using CSS `--variables` rather than these SCSS vars

Reply:
Potentially OOS